.reaction/config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Nope, it was gitignored (likely during the time work on this branch was happening)